data/yam/autoyast/create_hdd_textmode_ppc64le.xml

```diff
@@ -0,0 +1,467 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
It is different for power kvm and power VM, so it is better to update the name, such as:
data/yam/autoyast/create_hdd_textmode_powervm.xml

Reply:
> It is different for power kvm and power VM, so it is better to update the name, such as: data/yam/autoyast/create_hdd_textmode_powervm.xml

As I know, Yast team will not test power + kvm backend any more, then you don't need to change the name.

or if kvm is still tested, I would suggest keeping a single autoyast file for kvm backend/spvm/powerhmc, there should be 2 differences for kvm backend:

1.  set the root disk to `/dev/vda`
2.  we don't need to enable sshd service in firewalld

Reply:
For power kvm we still need it for yam migration test. For this ticket, we have a power vm profile is enough for current yast job group test. So I think we can rename it to powervm for now, for power kvm we can work on it on another ticket, and it should be for migration test on SLE12SP5.